page-objects/pages/TASKListPage.js

```diff
@@ -103,55 +103,18 @@ async function sortTasksCourse(courseName) {
 	await waitHelpers.waitUntilPageLoads();
 }*/
 
-async function getListOfTaskTitles() {
-	await waitHelpers.waitUntilElementIsNotVisible('.loaded #MathJax_Message');
-	return await elementHelpers.getTextFromAllElements(taskTitleContainer);
-}
-
-async function getListOfTask() {
-	await waitHelpers.waitUntilElementIsNotVisible('.loaded #MathJax_Message');
-	return elementHelpers.getListOfAllElements(taskContainer);
-}
-
-async function clickOnTask(button) {
+async function clickActionFromMenuOnTask(button) {
```

Comment:
You've cleaned up a lot of methods, but the selectors are still in place. I think we can remove them as well, or?
e.g. taskTitleContainer, taskContainer, ...

Reply:
Removed unused selectors